arm-eventhub/2017-04-01/swagger/EventHub.json

```diff
@@ -1661,6 +1666,71 @@
       ],
       "description": "Single item in List or Get Event Hub operation"
```

Comment:
Sounds as if this type is only part of responses. If that is correct and also supposed to stay that way in the future, please mark the new property and all properties of the 2 new data types as `readOnly`. (I'd suggest the same for other properties of `Eventhub`, but that would be a breaking change)